x-pack/test/functional/apps/maps/group3/reports/index.ts

```diff
@@ -63,7 +64,8 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
       await PageObjects.dashboard.navigateToApp();
       await PageObjects.dashboard.loadSavedDashboard('Ecommerce Map');
-      await PageObjects.reporting.openPngReportingPanel();
+      await PageObjects.reporting.openExportTab();
+      await testSubjects.click('pngV2-radioOption');
```

Comment:
Could this be helper function in the `reporting` page object? Something as simple as `PageObjects.reporting.clickPngRadioButton`, or perhaps as an option to `openExportTab`. That way, if the `data-test-subj` gets changed at some point in the future, it would only have to be changed **one place** rather than all over :)